app/presenters/vm_presenter.rb

```diff
@@ -0,0 +1,12 @@
+class VmPresenter
+  def initialize(controller, session)
+    @controller = controller
```

Comment:
Why do we pass in the controller into this class if we do not use it?  Is that some sort of pattern for presenters?